packages/quantic/package.json

```diff
@@ -28,11 +28,15 @@
     "e2e:watch": "cypress open --browser chrome --e2e",
     "e2e": "cypress run --browser chrome",
     "e2e:detailed": "cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
-    "scratch:create": "sf org create scratch --set-default --definition-file config/project-scratch-def.json --alias \"LWC\"",
-    "setup:examples": "../../node_modules/.bin/ts-node scripts/build/deploy-community.ts",
-    "deploy:main": "sf project deploy start --target-org LWC --source-dir force-app/main",
-    "deploy:examples": "sf project deploy start --target-org LWC --source-dir force-app/examples",
-    "scratch:dev": "npm run scratch:create && npm run deploy:main && npm run setup:examples",
+    "e2e:playwright": "npx playwright test",
+    "e2e:playwright:lws-enabled": "npx playwright test --project=LWS-enabled",
+    "e2e:playwright:lws-disabled": "npx playwright test --project=LWS-disabled",
+    "scratch:create": "sf org create scratch --set-default --definition-file config/lws-enabled-scratch-def.json --alias Quantic__LWS_enabled",
+    "setup:examples": "npm run create:lws-enabled && npm run create:lws-disabled",
+    "create:lws-enabled": "../../node_modules/.bin/ts-node scripts/build/deploy-community.ts --scratch-org-def-path=./config/lws-enabled-scratch-def.json",
+    "create:lws-disabled": "../../node_modules/.bin/ts-node scripts/build/deploy-community.ts --scratch-org-def-path=./config/lws-disabled-scratch-def.json",
+    "deploy:main": "sf project deploy start --source-dir force-app/main --target-org",
+    "deploy:examples": "sf project deploy start --source-dir force-app/examples --target-org",
```

Comment:
do you not have to specify the target-org aliases here? does it take whichever is already configured?

Reply:
having an npm script like this:
```
"deploy:examples": "sf project deploy start  --target-org"
```
means that this script accepts a parameter called  `target-org`

and you can call the npm script as the following to pass any alias you want:
```
npm run deploy:examples -- --target-org MY_ALIAS
```

it will be eqauivalent to executing:
```
sf project deploy start  --target-org MY_ALIAS
```